### DISCUSSION TITLE
Cant connect mac m1 using Samsung tablet as second display

### DISCUSSION THREAD

**Discussion by azagupta** [pavlobu/deskreen#268] [Bugs]
Im using a m1 mac when i open deskreen and put the ip address on my Samsung tablet it keeps saying site cannot be reached, they are on the same network and i have the dummy hdmi set up on my mac does anyone know why this might be?

**Comment by TimTools** [comment]
I have the same issue, it works when I disable default macos firewall. However I'm not willing to go that road.
